DMTR-121.tex

```diff
@@ -40,11 +40,46 @@ \section{Introduction}
 \subsection{Objectives}
 \label{sect:objectives}
 
+To acquire data from the spectrograph instrument along with proper
+headers and transfer all data to NCSA for further viewing in LSP.~~
+
 
 
 \subsection{System Overview}
 \label{sect:systemoverview}
 
+The spectragraph instrument is the first device to have real data for
+viewing and evaluation. ~ ~In the beginning, the image data from a new
+instrument requires human viewing before setting the future of an image.
+~The image could be deleted or be stored for further processing. ~
+Therefore the image data will be sent to NCSA for further viewing
+through the LSST Science Platform (LSP). ~ ~If further processing or the
+image data is to be kept for further analysis, it will be ingested into
+the DBB along with the image's metadata/provenance for further
+processing. ~ Version 1.0 of this test will have data written from the
+spectragraph directly to a local disk on the ATS system which is
+co-located and directly connected to the spectragraph instrument. ~ Data
+will be copied from that local disk on the ATS system to NCSA and a
+ButlerG2 ingest process completed so that the LSP can access the image
+data. ~ Version 2.0 of this test, when the spectragraph is on the
+summit, will be that all data from the spectragraph will be written on
+the L1 handoff system which will be at the BDC (base data center). ~The
+L1 handoff system will have the OODS service and the DBB service in
+place. ~The DBB service will transfer the data to NCSA and use the
+header files to generate metadata/provenance data. ~The DBB will be
+responsible to generate the ButlerG3 access files along with protect the
+data from never loosing it, and ingest it into the permanent record of
```

Comment:
-- deletion: not just deletion, but from file system corruption and other disaster circumstances that would cause the file to be lost.   So I can add some words to that.    I will also fix the butler (G2/G3) because it really depends on what folks need to use as to where the file is "ingested" into.